services/app-web/src/pages/QuestionResponse/QuestionResponseSummary/RateQuestionResponse.tsx

```diff
@@ -47,18 +51,31 @@ export const RateQuestionResponse = () => {
         fetchPolicy: 'network-only',
     })
 
+    const rate = data?.fetchRate.rate
     useEffect(() => {
         updateHeading({ customHeading: rateName })
     }, [rateName, updateHeading])
 
+    const {
+        data: fetchContractData,
+        loading: loadingContract,
+        error: fetchContractError,
+    } = useFetchContractQuery({
```

Comment:
Suggestion: This works, but I wonder if the change here is to add the `consolidatedStatus` in the response data of `fetchRateQuestions` fragments. It seems like upcoming features we will need to know more about the review status of the related contracts or rates.

Maybe in `contractRevisionSchema` we add in the `consolidatedStatus` so then reach contract revision has the status so we don't need to make another fetch.

Reply:
@JasonLin0991  Fair point. That's something I considered. The issue I saw with that is managing the `consolidatedStatus` on each rev. It seemed like a bigger change than I should independently put into a PR. I don't love needing to do 2 API calls here. I was thinking we might want something like  a `fetchRateWithContract` endpoint. That would remove the need to track `consolidatedStatus` on each rev and any cascading schema changes. I can bring this up in an RSP